nano/node/confirmation_solicitor.cpp

```diff
@@ -70,7 +69,7 @@ bool nano::confirmation_solicitor::add (nano::election const & election_a)
 		if (!exists || !is_final || different)
 		{
 			auto & request_queue (requests[rep.channel]);
-			if (request_queue.size () < max_channel_requests)
+			if (!rep.channel->full ())
```

Comment:
Completely filling up the buffers is possibly not a good idea. It might starve other processes of network bandwidth. How about half or quarter full?